src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,84 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Arrays;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private Object[] keyObjects = new Object[ARRAY_SIZE];
```

Comment:
initialize in constructor

```suggestion
    private K[] keys;
```